README.md

```diff
@@ -28,3 +33,23 @@ See [Code of Conduct](CODE_OF_CONDUCT.md).
 
 This software was developed by the National Snow and Ice Data Center with funding from
 multiple sources.
+
+
+## Development
```

Comment:
We do have a `doc/contributing.md` file that covers this. Maybe we should have a section in the README that points to "other docs"? I put that file in a separate dir because I like keeping the README as a concise entrypoint to the docs.

There is some OBE stuff about modelines there that we can get rid of.

Reply:
Ahha! I figured such a document would be in the project root, and I didn't see one, so I naturally figured it didn't exist. My mistake. I'll link to that document.